templates/controllers/tab/settings/productionStage/form/productionStageForm.tpl

```diff
@@ -46,4 +44,3 @@
 		{fbvFormButtons id="productionStageFormSubmit" submitText="common.save" hideCancel=true}
 	{/if}
 </form>
-<p><span class="formRequired">{translate key="common.requiredField"}</span></p>
```

Comment:
Just look into the template code, the first two elements have required=true, and it makes sense for them to be required. I think that the mark is not appearing because their respective form sections are not assigned to them by the "for" statement... but that's just a guess.